packages/codegen/templates/app00/domains/condo/constants.js

```diff
@@ -0,0 +1,14 @@
+const APP_TOKEN_KEY = 'appToken'
+const CONDO_ACCESS_TOKEN_KEY = 'condoAccessToken'
+const CONDO_REFRESH_TOKEN_KEY = 'condoRefreshToken'
+const CONDO_ORGANIZATION_ID_KEY = 'condoOrganizationId'
+
+const ACCEPT_LANGUAGE = 'accept-language'
```

Comment:
if you specified these constants in env, then you should use them from there. They should be the same